src/main/model/TypeOfBusiness.ts

```diff
@@ -0,0 +1,7 @@
+export enum TypeOfBusiness {
+    LIMITED_COMPANY = "Limited company",
```

Comment:
We need to make sure these enums are getting used corresponding controllers/views otherwise there will be multiple instances of same values and it will result in type mismatch

Reply:
Thats correct. This will be done as a refactor exercise where we will start using MongoDB for saving the data